scripts/deploy_docker.sh

```diff
@@ -0,0 +1,27 @@
+sh ignite.sh
+sleep 30
+export IGNITE_HOME=/root/git/apache-ignite-2.8.1-bin/
+docker stop -f $(docker ps |grep -v rally|awk '{print $1}'|grep -v CON)
+
+docker run --rm --net=host --name mm -p 9005:9005 -v /tmp:/tmp -itd mm
```

Comment:
For each service, we have the naming conversion, could you update the script by referencing to the doc https://github.com/xieus/alcor/blob/docs/design/docs/modules/ROOT/controller.adoc?

Reply:
fixed in 3bfa97f